### PR TITLE
Improve PR description code highlighting

### PR DESCRIPTION
**Pull request by itschrishudson**
Resolves https://github.com/brave/devops/issues/6517

Adds code block characters around file lists for the diffs to improve readability and syntax highlighting.

### CODE REVIEW

**Review by antonok-edm — APPROVED** [review]

File: .github/workflows/sync-from-fork.yml [hunk 1]
```diff
@@ -51,10 +51,14 @@ jobs:
         run: |
           git remote add upstream ${UPSTREAM_URL}
           git fetch upstream
-          git diff --diff-filter AM HEAD..remotes/upstream/master -- assets/resources/scriptlets.js >> changes.txt && echo >>changes.txt
-          git diff --diff-filter AM HEAD..remotes/upstream/master -- src/web_accessible_resources >> changes.txt && echo >>changes.txt
-          git diff --diff-filter AM HEAD..remotes/upstream/master -- src/js/redirect-engine.js >> changes.txt && echo >>changes.txt
+
+          FILES_TO_CHECK=("assets/resources/scriptlets.js"  "src/web_accessible_resources"  "src/js/redirect-engine.js")
+          for files in ${FILES_TO_CHECK[*]}; do
+          echo '```' >> changes.txt;
```
Comment (antonok-edm) [inline .github/workflows/sync-from-fork.yml:57]: Oh, actually - could you make this `'```diff'` so it gets syntax highlighting as well?

Comment (itschrishudson) [reply .github/workflows/sync-from-fork.yml:57]: Yep updated.

**Review by antonok-edm — APPROVED** [review]

**Comment by antonok-edm** [comment]
@pes10k this is gonna need your review

**Review by pes10k — APPROVED** [review]
lgtm

**Pull request merged**